stream-compositions/services/legal-entity-composition-service/pom.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.backbase.buildingblocks</groupId>
+        <version>13.4.0</version>
+        <artifactId>service-sdk-starter-core</artifactId>
+        <relativePath/>
+    </parent>
+
+    <artifactId>legal-entity-composition-service</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+
+    <packaging>jar</packaging>
+
+    <name>Stream :: Compositions :: Services :: Legal Entity</name>
+
+    <properties>
+        <boat-maven-plugin.version>0.15.0</boat-maven-plugin.version>
+    </properties>
+
+    <dependencies>
+        <!-- Service SDK dependencies -->
+        <dependency>
+            <groupId>com.backbase.buildingblocks</groupId>
+            <artifactId>events</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.backbase.buildingblocks</groupId>
+            <artifactId>auth-security</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.mapstruct</groupId>
+            <artifactId>mapstruct-processor</artifactId>
+        </dependency>
+
+        <!-- Streams dependencies -->
+        <dependency>
+            <groupId>com.backbase.stream.compositions.events</groupId>
+            <artifactId>legal-entity-events</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.backbase.stream.compositions.api</groupId>
+            <artifactId>legal-entity-composition-api</artifactId>
```

Comment:
I believe we don't need this guy here since it is already being handled by the maven-dependency-plugin `artifactItem` below